src_erl/Communication_Layer/http_Nerlserver/src/source/sourceStatem.erl

```diff
@@ -205,9 +205,11 @@ roundRobin(ListOfSamples,CSVPath,Counter,[{ClientName,WorkerName,RouterHost,Rout
 sendSample([Head|ListOfSamples],CSVPath,Counter,ClientName,WorkerName,RouterHost,RouterPort)->
 %%  io:format("CSVPath ~p ,Counter ~p ,[{ClientName ~p ,WorkerName ~p ,RouterHost ~p ,RouterPort ~p ~n",[CSVPath,Counter,ClientName,WorkerName,RouterHost,RouterPort]),
 %%  io:format("CSVPath ~p ~n",[list_to_binary([list_to_binary([list_to_binary(atom_to_list(ClientName)),<<"#">>,list_to_binary(WorkerName),<<"#">>,list_to_binary(CSVPath),<<"#">>,list_to_binary(integer_to_list(Counter)),<<"#">>]),list_to_binary(Head)])]),
-
-  http_request(RouterHost, RouterPort,"weightsVector",
-    list_to_binary([list_to_binary([list_to_binary(atom_to_list(ClientName)),<<"#">>,list_to_binary(WorkerName),<<"#">>,list_to_binary(CSVPath),<<"#">>,list_to_binary(integer_to_list(Counter)),<<"#">>]),list_to_binary(Head)])),
+  ToSend = term_to_binary({ClientName, WorkerName, CSVPath, Counter, Head}),
+  http_request(RouterHost, RouterPort,"weightsVector",ToSend),
+%%  http_request(RouterHost, RouterPort,"weightsVector",
```

Comment:
What are these commented lines? 
If needed then put some comment here otherwise you may clean it.